projects/fusion-ui/components/button/v4/button.component.scss

```diff
@@ -0,0 +1,232 @@
+@import '../../../src/style/scss/v4/colors';
+@import '../../../src/style/scss/v4/spacings';
+@import '../../../src/style/scss/v4/border-radius';
+@import '../../../src/style/scss/v4/vars/fonts';
+
+
+@mixin setButtonColor($color, $bg-color, $bg-color-hover, $outline-color, $outline-bg-color-hover){
```

Comment:
We don't want to put it at mixin style lib?

Reply:
This mixin relevant for the button component only.